src/components_1/ListOfCards.js

```diff
@@ -1,17 +1,24 @@
 import * as React from 'react';
+
 import MyCard from '../components_2/MyCard';
+
 import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
 import { CardActionArea } from '@mui/material';
 import List from '@mui/material/List';
 
-const ListOfCards = () => {
+const ListOfCards = () => (
     <List>
         <MyCard/>
         <MyCard/>
         <MyCard/>
+        {/* <MyCard 
```

Comment:
It appears that you are using it correctly, I think your restructuring was just a bit off.